hack/docker-builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:27
+FROM fedora:28
 
 ENV LIBVIRT_VERSION 3.7.0
```

Comment:
Change libvirt version, I think under Fedora28 4.2 is the latest version from Fedora repos

Reply:
4.1 it is according to jkoji